src/Bicep.Core.Samples/Files/InvalidResources_CRLF/Completions/deploymentScriptTopLevel.json

```diff
@@ -1,157 +1,157 @@
-      "newText": "resource"
-    }
-  },
-  {
-    "label": "resource",
-    "kind": "snippet",
-    "detail": "Nested resource with defaults",
-    "documentation": {
-      "kind": "markdown",
-      "value": "```bicep\nresource Identifier 'Type' = {\n  name: \n  properties: {\n    \n  }\n}\n```"
-    },
-    "deprecated": false,
-    "preselect": false,
-    "sortText": "2_resource",
-    "insertTextFormat": "snippet",
-    "insertTextMode": "adjustIndentation",
-    "textEdit": {
-      "range": {},
-      "newText": "resource ${1:Identifier} '${2:Type}' = {\n  name: $3\n  properties: {\n    $0\n  }\n}"
-    }
-  },
-  {
-    "label": "resource",
-    "kind": "snippet",
-    "detail": "Nested resource without defaults",
-    "documentation": {
-      "kind": "markdown",
-      "value": "```bicep\nresource Identifier 'Type' = {\n  name: \n  \n}\n\n```"
-    },
-    "deprecated": false,
-    "preselect": false,
-    "sortText": "2_resource",
-    "insertTextFormat": "snippet",
-    "insertTextMode": "adjustIndentation",
-    "textEdit": {
-      "range": {},
-      "newText": "resource ${1:Identifier} '${2:Type}' = {\n  name: $3\n  $0\n}\n"
-    }
-  },
-  {
-    "label": "tags",
-    "kind": "property",
-    "detail": "tags",
-    "documentation": {
-      "kind": "markdown",
-      "value": "Type: `Dictionary<string,String>`  \nResource tags.  \n"
-    },
-    "deprecated": false,
-    "preselect": false,
-    "sortText": "2_tags",
-    "insertTextFormat": "plainText",
-    "insertTextMode": "adjustIndentation",
-    "textEdit": {
-      "range": {},
-      "newText": "tags"
-    },
-    "commitCharacters": [
-      ":"
-    ]
-  }
+[
+  {
+    "label": "dependsOn",
+    "kind": "property",
+    "detail": "dependsOn",
+    "documentation": {
+      "kind": "markdown",
+      "value": "Type: `(module[] | (resource | module) | resource[])[]`  \nWrite-only property  \n"
+    },
+    "deprecated": false,
+    "preselect": false,
+    "sortText": "2_dependsOn",
+    "insertTextFormat": "plainText",
+    "insertTextMode": "adjustIndentation",
+    "textEdit": {
+      "range": {},
+      "newText": "dependsOn"
+    },
+    "commitCharacters": [
+      ":"
+    ]
+  },
+  {
+    "label": "identity",
+    "kind": "property",
+    "detail": "identity",
+    "documentation": {
+      "kind": "markdown",
+      "value": "Type: `ManagedServiceIdentity`  \nManaged identity generic object.  \n"
+    },
+    "deprecated": false,
+    "preselect": false,
+    "sortText": "2_identity",
+    "insertTextFormat": "plainText",
+    "insertTextMode": "adjustIndentation",
+    "textEdit": {
+      "range": {},
+      "newText": "identity"
+    },
+    "commitCharacters": [
+      ":"
+    ]
+  },
+  {
+    "label": "location",
+    "kind": "property",
+    "detail": "location (Required)",
+    "documentation": {
+      "kind": "markdown",
+      "value": "Type: `string`  \nThe location of the ACI and the storage account for the deployment script.  \n"

```

Comment:
My previous deploy-time constant fix PR has a regression which removes the description and this one fixes it.